src/dbclient.py

```diff
@@ -40,10 +43,10 @@ def insert_item(self, item: Item, autocommit=True):
             item.link,
             sanitize_string_for_database(item.brand) if item.brand else None,
             sanitize_string_for_database(item.model) if item.model else None,
-            item.price,
+            0 if item.price == None else item.price,
```

Comment:
```suggestion
            item.price or 0,
```

Reply:
you can test this

Reply:
proper for sure:
```
0 if item.price is None else item.price,
```

Reply:
```
0 if not item.price else item.price,
```

Reply:
https://stackoverflow.com/questions/13710631/is-there-shorthand-for-returning-a-default-value-if-none-in-python